### PR TITLE
Feat/gitlab runner update

### PR DESCRIPTION
**Pull request by markcsims**
https://trello.com/c/uVxSCaxd/1709-gitlab-runner-connection-error

### CODE REVIEW

**Review by peterstarling — CHANGES_REQUESTED** [review]

File: ci/gitlab-runners/main.tf [hunk 1]
```diff
@@ -54,12 +54,17 @@ module "gitlab-runner" {
   runners_gitlab_url       = var.gitlab_url
   enable_runner_ssm_access = true
 
+  # set this to some unique value if you want to set up runners in 2 or more regions
+  cache_bucket_prefix = "aus"
```
[the remark this answers is not in the document]
Comment (markcsims) [reply ci/gitlab-runners/main.tf:57]: This is not to do with the stack but rather the cache the runners are using. I've removed the comment.

File: ci/gitlab-runners/main.tf [hunk 2]
```diff
@@ -54,12 +54,17 @@ module "gitlab-runner" {
...
+
   gitlab_runner_security_group_ids = [data.aws_security_group.default.id]
 
   docker_machine_download_url   = "https://gitlab-docker-machine-downloads.s3.amazonaws.com/v0.16.2-gitlab.10/docker-machine-Linux-x86_64"
-  docker_machine_spot_price_bid = "0.06"
+  docker_machine_spot_price_bid = "0.09"
```
[the remark this answers is not in the document]
Comment (markcsims) [reply ci/gitlab-runners/main.tf:62]: The default is "on-demand-price", we can change to this?

File: ci/gitlab-runners/main.tf [hunk 3]
```diff
@@ -54,12 +54,17 @@ module "gitlab-runner" {
...
+  docker_machine_instance_type = "m5.xlarge"
```
[the remark this answers is not in the document]
Comment (markcsims) [reply ci/gitlab-runners/main.tf:63]: Moving to m5.xlarge makes it slightly faster. I looked at a sample builds from a couple of weeks ago and compared them to today. For a PR they were taking over 16 minutes, now we are taking over 14 minutes.

File: ci/gitlab-runners/main.tf (hunk 2 above)
[the remark this answers is not in the document]
Comment (markcsims) [reply ci/gitlab-runners/main.tf:62]: Sets an upper limit - leaving as is

**Review by peterstarling — APPROVED** [review]

**Pull request merged**